src/main/python/main/ayab/audio.py

```diff
@@ -20,73 +20,43 @@
 """Standalone audio player."""
 
 from __future__ import annotations
-import logging
 from os import path
 
 import simpleaudio as sa
-import wave
 
-from PySide6.QtCore import QObject, QThread
-from typing import TYPE_CHECKING, Optional
+from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
     from .ayab import GuiMain
 
 
-class AudioWorker(QObject):
-    def __init__(self, parent: GuiMain):
-        super().__init__()
-        self.__dir = parent.app_context.get_resource("assets")
-        self.__prefs = parent.prefs
+class AudioPlayer:
+    def __init__(self, gui: GuiMain):
+        self.__dir = gui.app_context.get_resource("assets")
+        self.__prefs = gui.prefs
         self.__cache: dict[str, sa.WaveObject] = {}
 
-    def play(self, sound: str, blocking: bool = False) -> None:
-        """Play audio and wait until finished."""
-        # thread remains open in quiet mode but sound does not play
+    def play(self, sound: str) -> None:
+        """Play audio."""
         if self.__prefs.value("quiet_mode"):
             return
         # else
         wave_obj = self.__wave(sound)
         if wave_obj is None:
             return
         # else
-        play_obj = wave_obj.play()
-        if blocking:
-            # wait until sound has finished before returning
-            play_obj.wait_done()
+        wave_obj.play()
 
-    def __wave(self, sound: str) -> Optional[sa.WaveObject]:
+    def __wave(self, sound: str) -> sa.WaveObject | None:
```

Comment:
Why this change?

Reply:
It is my understanding that `T | None` is now the recommended syntax over the older `Optional[T]`, e.g. see https://stackoverflow.com/a/69440627 .

I think I changed this because my IDE suggested it. I can revert this change (and the one below) if you don't like it, or if you prefer to upgrade all occurrences of `Optional` in a separate PR.